app/components/branch-row.js

```diff
@@ -19,11 +19,10 @@ export default Component.extend({
   isTriggering: false,
   hasTriggered: false,
 
-  commitUrl: computed('branch.repository.{slug,vcsType}', 'branch.last_build.commit.sha', function () {
+  commitUrl: computed('branch.repository.{slug,vcsType}', 'branch.last_build.commit.sha', 'vcsType', function () {
```

Comment:
It seems like the `repository.vcsType` is not required as a dependency anymore, right?
```suggestion
  commitUrl: computed('branch.repository.slug', 'branch.last_build.commit.sha', 'vcsType', function () {
```